pkg/kv/batch.go

```diff
@@ -754,3 +754,27 @@ func (b *Batch) addSSTable(
 	b.appendReqs(req)
 	b.initResult(1, 0, notRaw, nil)
 }
+
+// migrate is only exported on DB.
+//lint:ignore U1001 unused
+func (b *Batch) migrate(s, e interface{}, targetVersion roachpb.Version) {
```

Comment:
This commit belongs into a separate PR that revamps https://github.com/cockroachdb/cockroach/pull/42822

Reply:
#57445.